test/cpp/jit/test_backend_compiler_lib.cpp

```diff
@@ -85,9 +85,9 @@ class BackendWithCompiler : public PyTorchBackendInterface {
         auto sub = instruction.substr(15);
         const_val = stod(sub);
       } else if (token == "aten::add") {
-        output_list.emplace_back(x.add_(h, const_val));
+        output_list.emplace_back(x.add(h, const_val));
```

Comment:
`add_` is in-place. Using `add` instead of `add_`, to avoid confusing for users when they are reusing the same inputs multiple times for testing purpose (like in the unit test code).